lib/platform/github/index.ts

```diff
@@ -199,14 +219,20 @@ export async function initRepo({
     hostType: PlatformId.Github,
     url: defaults.endpoint,
   });
-  config.isGhe = URL.parse(defaults.endpoint).host !== 'api.github.com';
   config.renovateUsername = renovateUsername;
   [config.repositoryOwner, config.repositoryName] = repository.split('/');
   let repo: GhRepo;
   try {
+    let infoQuery = repoInfoQuery;
+    if (
+      config.isGhe &&
+      (!config.gheVersion || semverLessThan(config.gheVersion, '3.1.8'))
```

Comment:
Where does version 3.1.8 come from? I couldn't find `autoMergeAllowed` in any GHE api docs

Reply:
Schema changelog: https://docs.github.com/en/github-ae@latest/graphql/overview/changelog?query=#schema-changes-for-2021-08-03
Release dates: https://enterprise.github.com/releases

I've decided it would be safe to choose these values, and maybe I'm wrong

Reply:
`3.2.0` lineage is new, so it doesn't make sense to test it, they all should be okay

Reply:
But I'm not sure about 2.x.x